roofit/roofitZMQ/src/ZeroMQPoller.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+ * ROOT
+ * Copyright (c) 2018, Patrick Bos
+ * Distributed under the terms of the BSD 3-Clause License.
+ *
+ * The full license is in the file LICENSE, distributed with this software.
+ */
+
+#include <iostream>
+
+#include "RooFit_ZMQ/ppoll.h"
+#include "RooFit_ZMQ/ZeroMQPoller.h"
```

Comment:
Please always include the header that you are implementing first to ensure header consistency. Same for the other files in `roofit/roofitZMQ/src`.

Reply:
Done in 0c0dc78. I will also try to keep it in mind for coming PRs.